src/Plugin/GraphQL/DataProducer/EntityDefinition/Fields.php

```diff
@@ -73,16 +82,21 @@ public static function create(ContainerInterface $container, array $configuratio
    * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
    *   The entity type manager service.
    *
```

Comment:
there should be no empty line between param doc comments.

Reply:
Good catch. Fixed.